example/template.yaml

```diff
@@ -16,7 +16,7 @@ Resources:
     Properties:
       CodeUri: HelloWorldFunction
       Handler: helloworld.App::handleRequest
-      Runtime: java8
+      Runtime: java11
```

Comment:
Does it make sense to add the Runtime to Globals?

Reply:
Sure, I can Move to globals